frontend/src/app/home/Connected.tsx

```diff
@@ -15,10 +16,12 @@ export function Connected() {
   const fetchPet = useCallback(async () => {
     if (!account?.address) return;
 
-    const [name, _, energyPoints, parts] = await provider.view({
-      function: `${process.env.NEXT_PUBLIC_CONTRACT_ADDRESS}::main::get_aptogotchi`,
-      type_arguments: [],
-      arguments: [account.address],
+    const [name, _, energyPoints, parts] = await aptosClient.view({
+      payload: {
+        function: `${NEXT_PUBLIC_CONTRACT_ADDRESS}::main::get_aptogotchi`,
+        typeArguments: [],
```

Comment:
it is optional so can be removed if not needed
```suggestion

```

Reply:
remove